src/test/integration/nft.media.e2e-spec.ts

```diff
@@ -0,0 +1,52 @@
+import {NftService} from "../../endpoints/nfts/nft.service";
+import Initializer from "./e2e-init";
+import {Test} from "@nestjs/testing";
+import {PublicAppModule} from "../../public.app.module";
+import {NftFilter} from "../../endpoints/nfts/entities/nft.filter";
+import {Constants} from "../../utils/constants";
+import {NftMediaService} from "../../queue.worker/nft.worker/queue/job-services/media/nft.media.service";
+import {Nft} from "../../endpoints/nfts/entities/nft";
+
+
+describe('Nft Media Service', () => {
+  let nftMediaService: NftMediaService;
+  let nftService: NftService;
+  let nftIdentifier: string;
+
+
+  beforeAll(async () => {
+    await Initializer.initialize();
+
+    const moduleRef = await Test.createTestingModule({
+      imports: [PublicAppModule],
```

Comment:
I think here you can import QueueWorkerModule, to test that instance of the application

Reply:
Fixed